packages/server/src/db/utils.js

```diff
@@ -45,9 +43,10 @@ const DocumentTypes = {
   USER_FLAG: "flag",
 }
 
-const ViewNames = {
-  LINK: "by_link",
-  ROUTING: "screen_routes",
+const ViewModes = {
+  ALL: "all",
+  AUTOMATION: "auto",
```

Comment:
Yeah, I thought about building this as a function that we could utilise for other stuff, but realistically it'll probably be modes per view, rather than an enumeration with all usable modes in it - will update.